mod/groups/classes/Elgg/Groups/Menus/Title.php

```diff
@@ -38,13 +38,16 @@ public static function register(\Elgg\Hook $hook) {
 				'text' => elgg_echo('groups:edit'),
 				'link_class' => 'elgg-button elgg-button-action',
 			]);
-			$result[] = \ElggMenuItem::factory([
-				'name' => 'groups:invite',
-				'icon' => 'user-plus',
-				'href' => elgg_generate_entity_url($group, 'invite'),
-				'text' => elgg_echo('groups:invite'),
-				'link_class' => 'elgg-button elgg-button-action',
-			]);
+			
+			if (elgg_is_active_plugin('invitefriends')) {
```

Comment:
the plugin you need to check is `friends` not `invitefriends`

Reply:
I don't think so.
What if I want to use 'Friends' plugin but not the invite friends feature?

Reply:
since the group feature only allows you to invite already existing users (which are your friends) it has nothing to do with the invite friends feature which sends an invitation to non users to become users

Reply:
so if the friends feature is disabled, you can't have friends to invite

Reply:
Right, it confused me